src/App.js

```diff
@@ -84,6 +100,21 @@ function App() {
     setTimeout(() => {
       setCopied(false);
     }, 2000);
+
+    sendEvent({
+      category: 'interaction',
+      action: `button`,
+      label: 'copy'
+    });
+  };
+
+  const onButtonRegen = (isKata) => {
+    reGenerate(isKata);
+    sendEvent({
+      category: 'interaction',
+      action: `button`,
+      label: 'regen'
+    });
```

Comment:
Ah, yes, this is good idea. Otherwise we won't be able to make a difference between `reGenerate` from initial load/toggle change or `reGenerate` from "Re-gen" click 👍